### ISSUE TITLE
Add cy.scrollTo() command

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#497]
Add command `cy.scrollTo()` to allow scrolling to a specific position in the window or in the element found in the previous command. The DOM element or window must be in a "scrollable" state prior to the scroll happening. 

Addresses #313 

Valid positions are `topLeft`, `top`, `topRight`, `left`, `center`, `right`, `bottomLeft`, `bottom`, and `bottomRight`.

Examples:
```js
// window will scroll 250px to the right
cy.scrollTo(250)

// window will scroll 250px to the right
cy.scrollTo("250px")

// window will scroll to 50% of it's width
cy.scrollTo("50%")

// window will scroll to center of window's height and center of window's width
cy.scrollTo("center")

// scrollable el ".infinite-list" will scroll 250px to the right
cy.get(".infinite-list").scrollTo("250px")

// window will scroll 250px to the right and 50px down
cy.scrollTo(250, 50)

// window will scroll 250px to the right and 50px down
cy.scrollTo("250px", "50px")

// window will scroll down 60% of window's width and 60% of window's height
cy.scrollTo("60%", "60%")

// window will scroll down to window's bottom and rightmost point
cy.scrollTo("bottomRight")

// window will scroll 250px to the right over 200ms (default is 0 ms)
cy.scrollTo("250px", {duration: 200})

// window will scroll 250px to the right with swing easing animation
cy.scrollTo("250px", {easing: "swing"})
```

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
